### PR TITLE
18 and 107 add seeders for menu and menu items

### PR DESCRIPTION
**Pull request by d-vucetic**

### CODE REVIEW

File: src/main/java/com/g1dra/foodmanager/factory/MenuItemFactory.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    public MenuItem createMenuItem(Menu menu, User user, LocalDate localDate) {
+        return MenuItem.builder()
+                .menu(menu)
+                .food(foodFactory.create(8, user))
+                .date(localDate)
+                .build();
+    }
+
+    // TODO extract to utility class
```
Comment (LjSamardzic) [inline src/main/java/com/g1dra/foodmanager/factory/MenuItemFactory.java]: Why not use the in-built Java method - datesUntil?

File: src/main/java/com/g1dra/foodmanager/factory/FoodFactory.java [hunk 2]
```diff
@@ -27,14 +26,13 @@ public List<Food> create(int count, User user) {
         return list;
     }
 
+    // TODO shouldn't save
```
Comment (LjSamardzic) [inline src/main/java/com/g1dra/foodmanager/factory/FoodFactory.java:29]: Why not save?

**Review by LjSamardzic — APPROVED** [review]

File: src/main/java/com/g1dra/foodmanager/factory/FoodFactory.java (hunk 2 above)
Comment (d-vucetic) [reply src/main/java/com/g1dra/foodmanager/factory/FoodFactory.java:29]: Repository layer responsibility is persistence. Factories and Builders just create the object.

Comment (LjSamardzic) [reply src/main/java/com/g1dra/foodmanager/factory/FoodFactory.java:29]: True

**Pull request merged**